tests/translator/input/error_api_with_openapi_definition_body_no_flag.yaml

```diff
@@ -0,0 +1,58 @@
+Globals:
+  Api:
+    Name: "some api"
+    CacheClusterEnabled: True
+    CacheClusterSize: "1.6"
+    Auth:
+      DefaultAuthorizer: MyCognitoAuth
+      Authorizers:
+        MyCognitoAuth:
+          UserPoolArn: !GetAtt MyUserPool.Arn
+    Variables:
+      SomeVar: Value
+
+Resources:
+  ImplicitApiFunction:
+    Type: AWS::Serverless::Function
+    Properties:
+      CodeUri: s3://sam-demo-bucket/member_portal.zip
+      Handler: index.gethtml
+      Runtime: nodejs4.3
+      Events:
+        GetHtml:
+          Type: Api
+          Properties:
+            Path: /
+            Method: get
+
+  ExplicitApi:
+    Type: AWS::Serverless::Api
+    Properties:
+      StageName: SomeStage
+      DefinitionBody:
+        openapi: 3.1.1
```

Comment:
I would rather that this just worked instead of being an error. This is absolutely valid syntax.

@brettstack had an earlier comment:
> Currently we fail if we don't detect "swagger," however, we know that some developers add both "openapi" and "swagger" to get around this. Do we think we will potentially break customers who do this workaround? If so, we should fail with error message saying they need to specify OpenApiVersion 3 (also add code comment with this reason)

This goes along with my other comment. This doesn't change the existing behavior, so I guess we can change this later if it seems like we can. As a developer though, if I see that "SAM Supports OpenApi 3", my immediate thought is that I can just use OpenApi 3 in the normal way.

Reply:
I agree. I am onboard with fixing this so we treat having "openapi" in the def-body without the OpenApiVersion flag specified as valid syntax

Reply:
Fixed!